src/pages/home/report/ReportActionItemParentAction.tsx

```diff
@@ -135,7 +133,6 @@ function ReportActionItemParentAction({
                                               // Pop the chat report screen before navigating to the linked report action.
                                               Navigation.goBack(ROUTES.REPORT_WITH_ID.getRoute(ancestor.report.parentReportID ?? '-1', ancestor.reportAction.reportActionID));
                                           }
-                                          Timing.start(CONST.TIMING.SWITCH_REPORT);
```

Comment:
Why do we have to remove this invocation?

Reply:
Same case as https://github.com/Expensify/App/pull/46966/files#r1711083558